app/assets/javascripts/application/states.js

```diff
@@ -1,245 +1,358 @@
+  }, {
+    name: 'CONNECTICUT',
+    value: 'Connecticut'
+  }, {
+    name: 'DELAWARE',
+    value: 'Delaware'
+  }, {
+    name: 'DISTRICT OF COLUMBIA',
+    value: 'District of Columbia'
+  }, {
+    name: 'FEDERATED STATES OF MICRONESIA',
+    value: 'Federated States of Micronesia'
+  }, {
+    name: 'FLORIDA',
+    value: 'Florida'
+  }, {
+    name: 'GEORGIA',
+    value: 'Georgia'
+  }, {
+    name: 'GUAM',
+    value: 'Guam'
+  }, {
+    name: 'HAWAII',
+    value: 'Hawaii'
+  }, {
+    name: 'IDAHO',
+    value: 'Idaho'
+  }, {
+    name: 'ILLINOIS',
+    value: 'Illinois'
+  }, {
+    name: 'INDIANA',
+    value: 'Indiana'
+  }, {
+    name: 'IOWA',
+    value: 'Iowa'
+  }, {
+    name: 'KANSAS',
+    value: 'Kansas'
+  }, {
+    name: 'KENTUCKY',
+    value: 'Kentucky'
+  }, {
+    name: 'LOUISIANA',
+    value: 'Louisiana'
+  }, {
+    name: 'MAINE',
+    value: 'Maine'
+  }, {
+    name: 'MARSHALL ISLANDS',
+    value: 'Marshall Islands'
+  }, {
+    name: 'MARYLAND',
+    value: 'Maryland'
+  }, {
+    name: 'MASSACHUSETTS',
+    value: 'Massachusetts'
+  }, {
+    name: 'MICHIGAN',
+    value: 'Michigan'
+  }, {
+    name: 'MINNESOTA',
+    value: 'Minnesota'
+  }, {
+    name: 'MISSISSIPPI',
+    value: 'Mississippi'
+  }, {
+    name: 'MISSOURI',
+    value: 'Missouri'
+  }, {
+    name: 'MONTANA',
+    value: 'Montana'
+  }, {
+    name: 'NEBRASKA',
+    value: 'Nebraska'
+  }, {
+    name: 'NEVADA',
+    value: 'Nevada'
+  }, {
+    name: 'NEW HAMPSHIRE',
+    value: 'New Hampshire'
+  }, {
+    name: 'NEW JERSEY',
+    value: 'New Jersey'
+  }, {
+    name: 'NEW MEXICO',
+    value: 'New Mexico'
+  }, {
+    name: 'NEW YORK',
+    value: 'New York'
+  }, {
+    name: 'NORTH CAROLINA',
+    value: 'North Carolina'
+  }, {
+    name: 'NORTH DAKOTA',
+    value: 'North Dakota'
+  }, {
+    name: 'NORTHERN MARIANA ISLANDS',
+    value: 'Northern Mariana Islands'
+  }, {
+    name: 'OHIO',
+    value: 'Ohio'
+  }, {
+    name: 'OKLAHOMA',
+    value: 'Oklahoma'
+  }, {
+    name: 'OREGON',
+    value: 'Oregon'
+  }, {
+    name: 'PALAU',
+    value: 'Palau'
+  }, {
+    name: 'PENNSYLVANIA',
+    value: 'Pennsylvania'
+  }, {
+    name: 'PUERTO RICO',
+    value: 'Puerto Rico'
+  }, {
+    name: 'RHODE ISLAND',
+    value: 'Rhode Island'
+  }, {
+    name: 'SOUTH CAROLINA',
+    value: 'South Carolina'
+  }, {
+    name: 'SOUTH DAKOTA',
+    value: 'South Dakota'
+  }, {
+    name: 'TENNESSEE',
+    value: 'Tennessee'
+  }, {
+    name: 'TEXAS',
+    value: 'Texas'
+  }, {
+    name: 'UTAH',
+    value: 'Utah'
+  }, {
+    name: 'VERMONT',
+    value: 'Vermont'
+  }, {
+    name: 'VIRGIN ISLANDS',
+    value: 'Virgin Islands'
+  }, {
+    name: 'VIRGINIA',
+    value: 'Virginia'
+  }, {
+    name: 'WASHINGTON',
+    value: 'Washington'
+  }, {
+    name: 'WEST VIRGINIA',
+    value: 'West Virginia'
+  }, {
+    name: 'WISCONSIN',
+    value: 'Wisconsin'
+  }, {
+    name: 'WYOMING',
+    value: 'Wyoming'
+  }]
```

Comment:
hmmm, I actually like this how it was..  it's easy to see where ABBREV ends and FULL starts